### ISSUE TITLE
[BUG] Default help description prefix not updating when using custom embeds

### ISSUE THREAD

**Issue by stijnb1234** [Kaktushose/jda-commands#24]
When I change the prefix of a guild from `&` to `mp!` with `getJDACommands().getGuildSettings().put(guild.getIdLong(), commandSettings);`, and then type `mp!help`, I will get an embed like this:

![image](https://user-images.githubusercontent.com/27051051/117442635-53dd3480-af37-11eb-9bff-b44bc252a4ae.png)

So, the prefix in the commands changes, but not in the description of the embed.

The defaultHelp part of the embed.json I'm using:
```json
{
  "defaultHelp": {
    "title": "Help",
    "description": "De volgende commando's zijn beschikbaar. Typ `{prefix}{helpLabel} <command>` om specifieke hulp te krijgen",
    "color": "#00ff00"
  },
}
```

Can you look into this issue for me? With the default embeds it's working fine.

**Comment by Kaktushose** [comment]
I wasn't able to reproduce the bug.
Could you please provide more context? E.g. when are you updating the commandSettings object? Which version of jda-commands are you using? etc

**Comment by stijnb1234** [comment]
I'm on v1.1.1 of JDA-Commands, with Java 11 and JDA v4.2.1_262.

And I update the prefix like this:
```java
Guild guild = ...;
String newPrefix = ...;

final CommandSettings commandSettings = getDefaultSettings();
commandSettings.setPrefix(newPrefix);
jdaCommands.getGuildSettings().put(guild.getIdLong(), commandSettings);

public static CommandSettings getDefaultSettings() {
  return new CommandSettings("/", true, true, true);
}
```

------

**To be clear, I have this problem ONLY when I set up a custom JSON embed.**

```java
.setEmbedFactory(new JsonEmbedFactory(new File("embeds_nl.json")))
```

**Comment by Kaktushose** [comment]
this bug was fixed with commit [94c6a6a](https://github.com/Kaktushose/jda-commands/commit/94c6a6ade5bd5bdcd4e1d06e07fb5d4aaf9ed044). I haven't build a new version yet because there are still some things on my todo list that I want to implement first. Use this in your pom and you should be fine:
```xml
<dependency>
    <groupId>com.github.Kaktushose</groupId>
    <artifactId>jda-commands</artifactId>
    <version>d65ef83f5a</version>
</dependency>
```
